lib/fixup.rb

```diff
@@ -93,7 +93,7 @@
       raise StandardError, 'Failed to replace pkg name...'.lightred unless @device[:installed_packages].any? { |elem| elem[:name] == pkg[:pkg_rename] }
       # Ok to write working device.json
       File.write "#{CREW_CONFIG_PATH}/device.json", JSON.pretty_generate(JSON.parse(@device.to_json))
-      puts "#{pkg[:pkg_name].capitalize} renamed to #{pkg[:pkg_rename].capitalize}".lightgreen
+      puts "#{pkg[:pkg_name].capitalize} renamed to #{pkg[:pkg_rename].capitalize}".lightgreen if @opt_verbose
```

Comment:
Isn't this the message we want to show without verbosity?

Reply:
The other message is telling which status has been detected, i.e., renamed vs deprecated.

Should I just revert this part of the change entirely?

Reply:
Reverted.